routing/simple.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Routing</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css"
+   integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+   crossorigin=""/>
+</head>
+<body>
+  <script src="https://unpkg.com/leaflet@1.5.1/dist/leaflet.js"
+integrity="sha512-GffPMF3RvMeYyc1LWMHtK8EbPv0iNZ8/oTtHPx9/cc2ILxQ+u905qIwdpULaqDkyBKgOaB57QTMg7ztg8Jm2Og=="
+crossorigin=""></script>
+  <!--script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script-->
+  <script src="http://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+  <script src="../polyline/decode.js"></script>
+
+  <div style="width:100%">
+  <div id="map" style="height: 1050px"></div>
+  Base URL: <input id="baseurl" placeholder="http://localhost:8002/"></input></br>
+  Valhalla Token: <input id="token" rows="1" cols="30" maxlength="100" wrap="hard" placeholder="Enter token."></input></br>
+  Request Body:</br><textarea id="request_body" rows="5" cols="100" maxlength="500" wrap="hard">{"costing":"auto"}</textarea></br>
+  <button type="button" id="route">Route</button>
+
+  </div>
+
+  <script>
+    ////////////////////// INITIALIZATION ////////////////////////
+
+    //make a map using osm tiles
+    const map = L.map('map').setView([40.2, -76.6], 10);
+    const baseLayer = L.tileLayer('http://b.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+      maxZoom: 19,
+      attribution: '&copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributers'
+    });
+    baseLayer.addTo(map);
+
+    //state
+    let geojson = null;
+    let inputs = [];
+
+    /////////////////// REQUEST PROCESSING //////////////////////
+
+    //do all the requests
+    async function fetch(json_body) {
```

Comment:
everything that comes in here is an array of valhalla request jsons, even if the array only has one element